internal/supervisor/installation.go

```diff
@@ -328,12 +331,12 @@ func (s *InstallationSupervisor) createClusterInstallation(cluster *model.Cluste
 
 	size, err := mmv1alpha1.GetClusterSize(installation.Size)
 	if err != nil {
-		logger.WithError(err).Error("invalid cluster installation size")
+		logger.WithError(err).Error("Invalid cluster installation size")
```

Comment:
Nitpick but I don't think these error messages should start with a capital since Go concatenates them together as substrings with colon

Reply:
Actually, the way the logging library works is the `err` is put in structured logging separate from the error string defined. For instance, this code

```
		logger.WithField("webhookURL", hook.URL).WithError(err).Error("Unable to send webhook")
```

produces this error

```
ERRO[2020-07-01T11:20:42-04:00] Unable to send webhook                        error="Post \"http://localhost:8065\": dial tcp [::1]:8065: connect: connection refused" installation=tgh8d8txijbeipqqnudiz3bw7a instance=8eu38nwszbripqjte6nbzd3q8a webhookEvent=stable webhookURL="http://localhost:8065"
```